api/v1alpha1/ratelimit_types.go

```diff
@@ -42,6 +52,26 @@ type GlobalRateLimit struct {
 	Rules []RateLimitRule `json:"rules"`
 }
 
+// LocalRateLimit defines local rate limit configuration.
+type LocalRateLimit struct {
+
+	// Envoy requires a default rate limit to be set for each route.
+	// The other option is to set the default rate limit to unit32 max and set the
+	// fill interval to a short period, like 1 second. This will effectively make
+	// the default rate limit "unlimited".
+	// See https://www.envoyproxy.io/docs/envoy/latest/configuration/http/http_filters/local_rate_limit_filter#using-rate-limit-descriptors-for-local-rate-limiting
+
+	// Default is the default rate limit if no rules match.
+	Default RateLimitValue `json:"default"`
```

Comment:
Envoy requires a default rate limit to be set for each route.
The other option is to set the default rate limit to unit32 max and set the fill interval to a short period, like 1 second. This will effectively make the default rate limit "unlimited".

See https://www.envoyproxy.io/docs/envoy/latest/configuration/http/http_filters/local_rate_limit_filter#using-rate-limit-descriptors-for-local-rate-limiting